src/lib/test/fetchAccount.test.js

```diff
@@ -0,0 +1,84 @@
+import { describe, it, expect, vi } from 'vitest';
+import { fetchAccount, fetchAccountBalances } from '../stellar/horizonQueries'; 
+// @ts-ignore
+import { Server, StrKey } from 'stellar-sdk';
+
+vi.mock('stellar-sdk', async (importOriginal) => {
+  const actual = await importOriginal(); 
+
+  return {
+    ...(typeof actual === 'object' && actual !== null ? actual : {}), 
+    Server: vi.fn().mockImplementation(() => ({
```

Comment:
I don't exactly know the right way to do this for vitest, but since `Server` class from the old version of the SDK is now `Horizon.Server`, that should "somehow" (??) be reflected here